### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by andyf-canva**
Key notes:
- Swift/C++ interoperability is still not fully featured, not supporting more complex types of macros available in C++. I have had to point the Swift Package to the C umbrella file instead.
- I have decided to exclude ThorVG's OpenGL and WebGPU renderer initially, as they will require us to link external dependencies. As we continue to build the Swift wrapper, this can be re-evaulated, particularly the OpenGL renderer. 
- I have added a base `swift` folder. This will contain all necessary swift files to help expose the Lottie capabilities of ThorVG initially.
- I have created an additional test target to help test this Swift code within the package.
- I have tested the integration with some sample code, and am able to render a Lottie successfully 🎉 

Also, once this is merged, I will tag the repo with a `0.1.0` tag for a release.

### CODE REVIEW

**Review by rlaguilar — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,94 @@ doc
 *.tvg
 *.tmp
 *.code-workspace
+
```
Comment (ray-canva) [inline .gitignore:10]: I assume you copied it from our `canva-ios` repo?

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "thorvg-swift",
```
Comment (ray-canva) [inline Package.swift:6]: what a great name!

File: README.md [hunk 3]
```diff
@@ -1,483 +1,4 @@
...
-![BinarySize](https://img.shields.io/badge/Size-200kb-blue)
-[![CodeFactor](https://www.codefactor.io/repository/github/hermet/thorvg/badge)](https://www.codefactor.io/repository/github/hermet/thorvg)
-<br>
-[![Build Ubuntu](https://github.com/thorvg/thorvg/actions/workflows/build_ubuntu.yml/badge.svg?branch=main&event=push)](https://github.com/thorvg/thorvg/actions/workflows/build_ubuntu.yml)
-[![Build Windows](https://github.com/thorvg/thorvg/actions/workflows/build_windows.yml/badge.svg?branch=main&event=push)](https://github.com/thorvg/thorvg/actions/workflows/build_windows.yml)
-[![Build MacOS](https://github.com/thorvg/thorvg/actions/workflows/build_macos.yml/badge.svg?branch=main&event=push)](https://github.com/thorvg/thorvg/actions/workflows/build_macos.yml)
-[![Build iOS](https://github.com/thorvg/thorvg/actions/workflows/build_ios.yml/badge.svg?branch=main&event=push)](https://github.com/thorvg/thorvg/actions/workflows/build_ios.yml)
-[![Build Android](https://github.com/thorvg/thorvg/actions/workflows/build_android.yml/badge.svg?branch=main&event=push)](https://github.com/thorvg/thorvg/actions/workflows/build_android.yml)
+# thorvg-swift
```
Comment (ray-canva) [inline README.md:1]: well thought out! nice one!

**Review by ray-canva — APPROVED** [review]
LGTM, can't wait to see some beautiful animations from this.

BTW, better to use squash PR merge

File: README.md (hunk 3 above)
Comment (ray-canva) [reply README.md:1]: Oh, maybe add the original repo version or commit SHA here or somewhere.

So it's obvious which ThorVG version is used in this fork.

Can link to `/src/bindings/capi/config.h` file in README file for a quick access to find out version too

File: .gitignore (hunk 1 above)
Comment (andyf-canva) [reply .gitignore:10]: Grabbed it from [here](https://github.com/github/gitignore/blob/main/Swift.gitignore).

File: README.md (hunk 3 above)
Comment (andyf-canva) [reply README.md:1]: Have added those details in the README 👍

**Pull request merged**